tests/manage/monitoring/pagerduty/test_ceph.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import managed_service_required, tier4, tier4a
+from ocs_ci.ocs import constants
+from ocs_ci.utility import pagerduty
+
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@tier4a
+@managed_service_required
+@pytest.mark.polarion_id("OCS-903")
```

Comment:
shouldn't we create a new Polarion test case for the pagerduty use case?

Reply:
Polarion IDs updated.